life_dashboard/tests/test_auth.py

```
@@ -0,0 +1,99 @@
+        self.selenium.find_element(By.CSS_SELECTOR, 'button[type="submit"]').click()
+        
+        # Wait for redirect to dashboard
+        WebDriverWait(self.selenium, 10).until(
+            EC.url_contains(reverse('dashboard'))
+        )
+        
+        # Verify we're logged in
+        assert 'Welcome' in self.selenium.page_source 
+
+
+if name == '__main__':
```

Ruff (0.11.9)

Undefined name `name`

(F821)